arex-instrumentation/dubbo/arex-dubbo/src/main/java/io/arex/inst/dubbo/DubboAdapter.java

```diff
@@ -59,11 +60,24 @@ public String getServiceOperation() {
         return getPath() + "." + getOperationName();
     }
     public String getRequest() {
-        Object request = null;
-        if (invocation.getArguments() != null && invocation.getArguments().length > 0) {
-            request = invocation.getArguments()[0];
+        return parseRequest(invocation.getArguments(), Serializer::serialize);
```

Comment:
>return parseRequest(invocation.getArguments(), Serializer::serialize);

How to determine which serializer is supported to use?

The serializer used by the user may be customized and not supported, such as JSON.

Reply:
customize serializer protocol this suite situation only arex-schedule replay need (specify the serialization implemet for the user through SPI during replaying)